printthread.php

```diff
@@ -20,6 +20,7 @@
 
 // Load global language phrases
 $lang->load("printthread");
+$lang->load("archive");
```

Comment:
It makes little sense to load that language file here.

Reply:
Are you sure?

Reply:
In any case, I would not expect this language file to be used anywhere outside the archive.

Reply:
I will revert this part for this PR.